src/binaryen-c.cpp

```diff
@@ -5451,7 +5451,10 @@ void BinaryenModuleRunPasses(BinaryenModuleRef module,
   PassRunner passRunner((Module*)module);
   passRunner.options = globalPassOptions;
   for (BinaryenIndex i = 0; i < numPasses; i++) {
-    passRunner.add(passes[i]);
+    passRunner.add(passes[i],
+                   globalPassOptions.arguments.count(passes[i]) > 0
+                     ? globalPassOptions.arguments[passes[i]]
+                     : std::optional<std::string>());
```

Comment:
Does it compile with `{}` or `std::nullopt` here? I'd prefer them in that order if possible, but if the compiler says no, we can leave it...